exercise_010_multiversal_equality/README.md

```diff
@@ -1,67 +1,36 @@
-# Opaque Type Aliases
+# Multiversal Equality
 
 ## Background
 
-An Opaque Type Alias can be used to provide the functionality of a "wrapper
-type" (i.e. a type that wraps, and therefore hides, another type) but without
-any runtime overhead. The aim is to provide additional type-safety at
-compile-time but then be stripped away at runtime. It is a powerful new feature
-of Scala 3 for supporting Information Hiding.
-
-Opaque Type Aliases differ from plain Scala 2 Type Aliases in that the later
-just provide a new name for a type but wherever this new name is used, the
-call-site still knows the details of the original type being aliased. With
-Opaque Type Aliases, the original type being aliased is hidden (or is opaque) at
-the call-site.
+Scala 2.x uses _universal equality_, meaning that any two values of any two
+types can be compared with each other using `==` and `!=`. However, this can
+sometimes lead to buggy programs where we assume that we are comparing two
+values of the same type but in fact the types are different. In this case,
+because of universal equality, the compiler validates the buggy code but at
+runtime the comparison will _always_ return the same result regardless of the two
+values being compared -- it will always return `false` in the case of `==`,  and
```

Comment:
Replaced "--" with "—". (Not 100% sure however how it will look like)

```suggestion
values being compared — it will always return `false` in the case of `==`,  and
```